src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -587,3 +587,10 @@ services:
             - '@=service("prestashop.adapter.form.choice_provider.profile_by_id_choice_provider").getChoices()'
         tags:
             - { name: form.type }
+
+    form.type.database.sql_request:
+        class: 'PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\RequestSql\SqlRequestType'
+        calls:
```

Comment:
maybe this should be auto wired via the `TranslatorAwareTrait` ?

Reply:
what do you mean? :thinking:

Reply:
this might not be the right PR to do it, but @mickaelandrieu suggested that we use auto wiring a bit more in order to simplify the service declarations (both for us and the community)
you can auto inject services with the `@required` annotation https://symfony.com/doc/current/service_container/autowiring.html#autowiring-other-methods-e-g-setters
so it could be nice to have this in the `TranslatorAwareTrait` but I think dealing with auto wiring is an issue of its own, we will deal with it later